packages/transaction-controller/src/TransactionController.ts

```diff
@@ -143,6 +145,11 @@ export const CANCEL_RATE = 1.5;
  */
 export const SPEED_UP_RATE = 1.1;
 
+/**
+ * Multiplier used to determine a transaction's post transaction balance update interval
```

Comment:
```suggestion
 * Interval in milliseconds between checks of post transaction balance 
```

Reply:
Fixed in https://github.com/MetaMask/core/pull/1877/commits/54ec5d3b7a2cef7e859ef1335c34fd8f64cd2e3d